migration/migrator/migrations/course/20200608034200_peer_feedback.py

```diff
@@ -0,0 +1,110 @@
+"""Migration for a given Submitty course database."""
+
+
+def up(config, database, semester, course):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    :param semester: Semester of the course being migrated
+    :type semester: str
+    :param course: Code of course being migrated
+    :type course: str
+    """
+    
+    database.execute(
+        """
+       DROP TABLE IF EXISTS peer_feedback;
+        """
+    )
+
+    # Create overall comment table
+    database.execute(
+        """
+       CREATE TABLE IF NOT EXISTS peer_feedback (
+            pf_id integer NOT NULL,
+            grader_id character varying(255) NOT NULL,
+            user_id character varying(255),
+            team_id character varying(255),
+            g_id character varying(255) NOT NULL,
+            feedback character varying(255),
+            CONSTRAINT user_team_id_check CHECK (user_id IS NOT NULL OR team_id IS NOT NULL)
+        );
+        """
+    )
+
+
+    database.execute("ALTER TABLE peer_feedback DROP CONSTRAINT IF EXISTS peer_feedback_pkey")
+    database.execute(
+        """
+        ALTER TABLE ONLY peer_feedback
+            ADD CONSTRAINT peer_feedback_pkey PRIMARY KEY (pf_id);
+        """
+    )
```

Comment:
Make the changes mentioned above to the migration as well

Reply:
Made the mentioned changes (removing pf_id and team_id)